### ISSUE TITLE
rs should be datable 

### ISSUE THREAD

**Issue by ebeshero** [TEIC/TEI#2389]
[Discussion on the TEI listserv of 25 January 2023](https://listserv.brown.edu/cgi-bin/wa?A1=ind2301&L=TEI-L#9) proposes that `<rs>` should be a member of att.datable.

**Comment by skurzinz** [comment]
The example that @KollatzThomas gave in the original post was 

> From time to time we would like to encode e.g. an event with a date-attribute: 
> `<rs type="event" subtype="conference" from="2023-09-03" to="2023-09-08">TEI conference</rs>`

Given the recent changes proposed in #2427, I would rather do something like 

```
<standOff><listEvent><event xml:id="TEIMEC2023" from="2023-09-03" to="2023-09-08">
<eventName type="alternative">TEI Conference 2023</eventName>
<eventName type="official">Encoding Cultures – Joint MEC and TEI Conference 2023</eventName>
</event></listEvent></standOff>
```
(if #2427 is not considered, replace `eventName` with `label` or `head`)

and point to that in the `<rs type="event" ref="#TEIMEC2023">TEI conference</rs>` – if a/the proper name for it were used it could then even be `<eventName ref="#TEIMEC2023">TEI conference</eventName>` (or `<name type="event">` if you want to stay at 4.6.0).

It is the *event* (just as the person/object/place/bibl/relation), not the *reference* (that paradigm would also include ptr/link/ref) that should be dated.

Hence opposed to the agreement in the previous discussion on TEI-L I would argue against making <rs> a member of `att.datable`.

**Comment by ebeshero** [comment]
In this particular instance of referencing a date by name, I tend to agree with @skurzinz . However, in the larger view, `<rs>` as a referencing string (rather than a named entity) might still be subject to date conditions. Can we imagine a different example to illustrate the temporality of such a string?

**Comment by ebeshero** [comment]
I am thinking about references to stages in a person's life that are not necessarily events:

```
<p>In <rs notAfter="1985-02-11">a time before her marriage</rs> she thought differently on the subject of household income.</rs>
```

I just made that up--perhaps we can think of better examples.

**Comment by skurzinz** [comment]
`CO-CoreElements.xml` says

> A referring string is a phrase which refers to some person, place, object, etc.

So the possible dating in question would only be used for temporalizing the validity of the string being a referring one. 

Made up example which I would argue against: 

```
<rs type="person" ref="#KarlI">Karl I</rs> was later not considered the <rs type="person" ref="#KarlI" notAfter="1918-11-11">emperor</rs> any longer.
```

If I were in the need to mark up this set of statements, I't say 

1. "emperor" when used as an office title with the person entity could be seen as an `org`, or I would not even mark it up in this case as it's about denomination and not about referring
2. the person stays the same, and its relations such as "rules over the Austrian part of the Habsburg monarchy from 1916 to this date" could be expressed in the `listPerson/person::descendant`s more clearly. If needed and no `@role` or other attribute was in place in my schema I'd say `<addName type="office" notAfter="1918-11-11">emperor</addName>`.^[1]

[1] Now there you got me: `addName` is not in `att.datable`  (as opposed to ` (pers|place|object|org)Name`  and `name`).

**Comment by skurzinz** [comment]
> `<p>In <rs notAfter="1985-02-11">a time before her marriage</rs>` 

If you split the person's life into spans you consider as separate states/phases, you could do that in `person/state` and link to these phases, but you don’t need the `att.datable` on the referring string. I was now thinking about what that phrase actually refers to: a phase in the life of her: 

```
<person>
   <persName>she</persName>
   <birth when="1965"/>
   <state notBefore="1965" notAfter="1985-02-11" xml:id="unmarriedher"><label>unmarried</label></state>
   <state notBefore="1985-02-12" xml:id="marriedher"><label>married</label></state>
   <event when="1985-02-11"><label>marriage</label></event>
</person>
```

Other than that, this could be expressed as 

`<p>In a time before her <date when="1985-02-11" type="implicit">marriage</date> [1]
or <rs type="event" ref="#marr">marriage</rs>`

the latter with the dating being part of the `<listEvent>` or `<person>` data? 

[1] Edge case here as the string itself is not mentioning a date, but a timespan. Yet there are examples for `date` in the ND chapter that mean timespans (my search term was `"century</rs>"`)

**Comment by lb42** [comment]
If you put a dating attribute on the naminf stribg you are takjibg about the dating of the name, i.e. when that name was used to refer to that person. That may or may not correspond to event dating of course

**Comment by joeytakeda** [comment]
In terms of an example, what about a phrase like "the before times" to mean sometime before early 2020 or so:  `<rs notBefore="2020">the before times</rs>` ?

**Comment by skurzinz** [comment]
@joeytakeda but this is again referring to a timespan -- which does not exist as an (entity-like) concept in TEI other than as a `<date from="" to="">`. 

Other than the missing starting point, what differentiates your example from "the 18th century" or "the 18th dynasty"?

**Comment by skurzinz** [comment]
(The implicit dating of an `<rs>` would be inferable from the `listChange` or the `publicationStmt`: This document refers to this thing (bei it an event, person, object or timespan) as of the date it was created or published.)

**Comment by sydb** [comment]
I am sorry, I am a bit confused here. Can someone explain to me what the problem is with this ticket? That is, I am wondering if there is any question that `<rs>` need be in att.datable? (I concede the point that some of the examples, above, are quite suspect. But I do not think that changes the fact that `<rs>` should be datable. After all, referring strings, like names, come and go over time; and any name could be encoded as `<rs>` rather than `<name>`.)

It seems to me the only discussion needed is “How do we make clear in the prose that the date (or date range) provided on the attributes refer to the date (or range) that the encoded string referred to the entity (either implied or explicitly indicated by `@ref` or `@key`), not that the entity itself occurred?”.

**Comment by sabineseifert** [comment]
Ok, adding `<rs>` to `att.datable` seems to be agreed upon.

As for the second question (How to make clear that it is about the dating of the name/referring string etc. and not about the date of the entity that is referred to): 
Perhaps add a note to `<rs>` (https://tei-c.org/release/doc/tei-p5-doc/en/html/ref-rs.html) (which is propably better than adding a second paragraph in the note of `att.datable`, https://tei-c.org/release/doc/tei-p5-doc/en/html/ref-att.datable.html)? Something like: 
Dating attributes on reference strings refer to the dating of the reference or name of the entity, e.g. when that reference or name was used to refer to the entity in question), not to the date of the entity itself.

**Comment by ebeshero** [comment]
Yes, @sabineseifert , I think that neatly explains the intended usage.

**Comment by sydb** [comment]
Tweaking @sabineseifert’s suggestion a bit, the following would be added to rs.xml:
~~~xml
<remarks versionDate="2023-11-08" xml:lang="en">
  <p>A dating attribute on a referring string is used to assert when the encoded
    word or phrase was used to refer to the entity in question, rather than when
    the entity existed.</p>
</remarks> 
~~~

I would also recommend an example. I commonly brag that I have been in the tallest building in the world 3 times, which made me think of the following.[1]
~~~xml
  <exemplum xml:lang="en" versionDate="2023-11-07">
    <p>The following example is a transcript of a fictional
    <soCalled>question</soCalled> in a Jeopardy-like game.</p>
    <egXML xmlns="http://www.tei-c.org/ns/Examples">
      <ab type="answer">It has been known as <rs from="1931-04-11"
      to="1973-04-04">the tallest building in the world</rs> and the
      <rs from="1973-04-04" to="1973-09-09">third tallest building in
      the world</rs>, but never the 2nd tallest building in the
      world.</ab>
      <ab type="question">What is the <name>Empire State Building</name>?</ab>
    </egXML>
  </exemplum>
~~~

**Note**
[1] As a child my family visited the Empire State Building in the late 1960s, when it was tallest in the world; in 1973 I went to the 88th floor of the World Trade Center, when it was the tallest building in the world (we did not go higher because of clouds above that level — we would not have been able to see anything, anyway); and in 2006 [Marcus Bingenheimer](https://liberalarts.temple.edu/about/faculty-staff/marcus-bingenheimer) took [Julia Flanders](https://cssh.northeastern.edu/faculty/julia-flanders/) and me up to the observation deck of Taipei 101, while it was the tallest building in the world.
![2006-03-09T07:02:10_mod04](https://github.com/TEIC/TEI/assets/124655/66e1fafb-162a-4d15-99ff-183bff632b7c)

**Comment by martindholmes** [comment]
Council FTF subgroup suggests we proceed with this, adding rs to att.datable and using the suggested example and remarks.

**Comment by ebeshero** [comment]
Council F2F in Vancouver: agrees to implement this, especially admiring the example.

**Comment by martindholmes** [comment]
Fix is merged.
